src/server/editorServices.ts

```diff
@@ -2083,9 +2098,27 @@ namespace ts.server {
 
             this.printProjects();
 
+            this.telemetryOnOpenFile(project);
             return { configFileName, configFileErrors };
         }
 
+        private telemetryOnOpenFile(project: ConfiguredProject | ExternalProject | undefined): void {
+            if (!this.eventHandler) return;
+            // Don't send for a project with 'checkJs' enabled globally.
+            if (project && project.getCompilationSettings().checkJs) return;
+
+            const stats: Mutable<OpenFileStats> = { js: 0, checkJs: 0 };
+            this.filenameToScriptInfo.forEach(scriptInfo => {

```

Comment:
We arent sending stats from start of the session to this point in time here. I thought that was the indent of this event. Do we need this info only for open files or all files ?